api/collections/serializers.py

```diff
@@ -143,7 +143,7 @@ def make_instance_obj(self, obj):
         return {'data': [
             pointer for pointer in
             obj.nodes_pointer
-            if not pointer.node.is_deleted and not pointer.node.is_folder
+            if not pointer.node.is_deleted and not pointer.node.is_collection
```

Comment:
Is the switch here from folder to collection (17829c5fb022bb67c79ea0d9318c706e18d84001, abad36ede5a98d1e10fd5373fd2d8a9417a1b0bd) strictly necessary for this PR? Or is this an extra clean-up while we're at it (cf. https://github.com/caneruguz/osf.io/pull/17#issuecomment-178075559)? Or ... ?